images/makeproject/project/onechunk_workgen.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+IFS=' '
+filename='tasks.txt'
+
+while read line; do
+    read -a strarr <<< "$line"
+    ./bin/create_work --appname onechunk \
+        --result_template templates/onechunk_out \
+        --remote_file "https://ssg.seedfinding.com/tasks/get?taskid=${strarr[0]}"\
+        "${strarr[1]}" \
+        "${strarr[2]}" \
+        --priority 12000 \
+        --min_quorum 2 \
+        --credit 2500 \
+        --delay_bound 259200
```

Comment:
Can we increase this to 1209600? 

I'm worried 3 days won't be enough for ARM devices that are shared across multiple projects.

2 weeks should be plenty and sounds about right for some of the other CPU projects I've seen on BOINC.
```suggestion
        --delay_bound 1209600
```